src/c2ffi/generator.lisp

```diff
@@ -537,7 +537,7 @@ target package."
   (with-standard-io-syntax
     (with-input-from-file (in c2ffi-spec-file :external-format (uiop:encoding-external-format :utf-8))
       (with-output-to-file (*c2ffi-output-stream* output :if-exists :supersede
-                            :external-format (uiop:encoding-external-format output-encoding))
+                                                         :external-format (uiop:encoding-external-format output-encoding))
```

Comment:
for `git blame` it's better to avoid recording changes to parts that you don't need to touch.

Reply:
Would it solve the problem if I change the indentation back to how it was and push again? Or would it be easier to make another pull request at this point?

Reply:
@attila-lendvai indentation restored, seems to have done the trick

Reply:
thanks!

FYI, you can `git push -f` into branches on github, and the relevant PR gets updated automatically.

Reply:
@attila-lendvai my bad, got it now!